src/components/views/settings/ChangePassword.tsx

```diff
@@ -140,7 +101,8 @@ export default class ChangePassword extends React.Component<IProps, IState> {
             phase: Phase.Uploading,
         });
 
-        cli.setPassword(authDict, newPassword).then(() => {
+        // This no longer logs out all sessions:
+        cli.setPassword(authDict, newPassword, false).then(() => {
```

Comment:
We can't assume this actually, we do not know if the connected server supports this API yet

Reply:
This is now dealt with by querying the spec versions that the server supports.